modules/lang-painless/src/main/java/org/elasticsearch/painless/antlr/Walker.java

```diff
@@ -257,7 +254,7 @@ public ANode visitSource(SourceContext ctx) {
             name = Character.toLowerCase(name.charAt(0)) + name.substring(1);
 
             statements.add(new SDeclaration(nextIdentifier(), location(ctx),
-                    new DResolvedType(location(ctx), scriptClassInfo.getGetReturns().get(index), false), name, false, null));
```

Comment:
What's happening here? (pinning for chat)

Reply:
`Class<?> type` moved into nodes that need it rather than adding nodes around the type.